cellpack/autopack/ingredient/multi_sphere.py

```diff
@@ -4,6 +4,7 @@
 
 from cellpack.autopack.ingredient.single_sphere import SingleSphereIngr
 import cellpack.autopack as autopack
+import numpy
```

Comment:
[Nit] and i'm not always great at this, but it's a nice convention to have third party imports first and local imports second